pkg/asset/installconfig/platform.go

```diff
@@ -70,15 +71,15 @@ func (a *Platform) queryUserForPlatform() string {
 func (a *Platform) awsPlatform() (*asset.State, error) {
 	return assetStateForStringContents(
 		AWSPlatformType,
-		asset.QueryUser(a.InputReader, "Region:"),
+		asset.QueryUser(a.InputReader, "Region:", "us-east-1"),
 	), nil
 }
 
 func (a *Platform) libvirtPlatform() (*asset.State, error) {
 	return assetStateForStringContents(
 		LibvirtPlatformType,
 		// TODO(yifan): Set the default URI.
-		asset.QueryUser(a.InputReader, "URI:"),
+		asset.QueryUser(a.InputReader, "URI:", "qemu:///system"),
```

Comment:
@crawford specifically rejected a default for this [here][1].  I dunno how he feels now.  I'm fine either way.

[1]: https://github.com/openshift/installer/pull/127#discussion_r210104585